packages/web/src/dev/ValidatorsList.tsx

```diff
@@ -264,19 +273,32 @@ class ValidatorsList extends React.PureComponent<Props, State> {
     }
 
     return (data || [])
-      .sort((a, b) => b.id - a.id)
       .sort((a, b) => compare(dAccessor(a), dAccessor(b)))
       .sort((a, b) => dir * compare(accessor(a), accessor(b)))
+      .sort((a, b) => this.isPinned(b) - this.isPinned(a))
+  }
+
+  isPinned({ address }: any) {
+    const list = (localStorage.getItem(localStoragePinnedKey) || '').split(',') || []
+    return +list.includes(address)
+  }
+
+  onPinned() {
+    this.setState({ update: Math.random() } as any)
```

Comment:
I dont quite follow this? 

it doesnt look like update is a property of State, and why a random value?

Reply:
I the solution that I found to force to sort the data again after pinning a group. The group emits an event and then is forced to sort again.